fastlane/Matchfile

```diff
@@ -3,7 +3,5 @@
 # This Matchfile has the shared properties used for all signing types
 
 # Store certs/profiles encrypted in Google Cloud
-storage_mode('google_cloud')
-google_cloud_bucket_name('a8c-fastlane-match')
-secrets_directory = File.join(Dir.home, '.configure', 'woocommerce-ios', 'secrets')
-google_cloud_keys_file(File.join(secrets_directory, 'google_cloud_keys.json'))
+storage_mode('s3')
+s3_bucket('a8c-fastlane-match')
```

Comment:
We'll need to add `s3_region` to address a runtime warning. See https://github.com/Automattic/simplenote-macos/pull/1150#discussion_r1591913764

```suggestion
s3_bucket('a8c-fastlane-match')
s3_region('us-east-2')
```